channeld/channeld.c

```diff
@@ -2003,6 +2003,9 @@ static void handle_tx_sigs(struct peer *peer, const u8 *msg)
 	struct witness_stack **ws;
 	const struct wally_tx *wtx;
 	size_t j = 0;
+	enum tx_role role = peer->channel->opener == REMOTE
+		? TX_INITIATOR : TX_ACCEPTER;
```

Comment:
This is *their role*, right?  If they opened the channel, they're the initiator...